packages/discord.js/src/structures/Poll.js

```diff
@@ -26,31 +47,22 @@ class Poll extends Base {
     /**
      * The media for a poll's question
      * @typedef {Object} PollQuestionMedia
-     * @property {string} text The text of this question
+     * @property {?string} text The text of this question
      */
 
     /**
      * The media for this poll's question
      * @type {PollQuestionMedia}
      */
     this.question = {
-      text: data.question.text,
+      text: data.question.text ?? null,
     };
 
-    /**
-     * The answers of this poll
-     * @type {Collection<number, PollAnswer>}
-     */
-    this.answers = data.answers.reduce(
-      (acc, answer) => acc.set(answer.answer_id, new PollAnswer(this.client, answer, this)),
-      new Collection(),
-    );
-
     /**
      * The timestamp when this poll expires
-     * @type {number}
+     * @type {?number}
      */
-    this.expiresTimestamp = Date.parse(data.expiry);
+    this.expiresTimestamp = data.expiry && Date.parse(data.expiry);
 
     /**
```

Comment:
Due to partials, I believe the `allowMultiselect` and `layoutType` updates below should be moved into `_patch()`